gef-ui/server/utils/deal-helpers.test.js

```diff
@@ -113,6 +114,21 @@ describe('makerCanReSubmit', () => {
 
     expect(result).toEqual(false);
   });
+
+  it('Should return false as the Maker is from a different Bank', () => {
```

Comment:
These tests were in the wrong file & have been moved (almost) verbatim from `helpers.test.js`